submissions/cuteman18/HTML_and_CSS_practice/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
```

Comment:
Metatag Viewport is mandatory for mobile markup.
https://serpstat.com/ru/blog/chto-takoe-metateg-viewport/
Could you please add it?
`<meta name="viewport" content="width=device-width, initial-scale=1.0">`

Reply:
Thanks, added